package.json

```diff
@@ -12,7 +12,7 @@
     "node": ">=16.18.0"
   },
   "scripts": {
-    "start": "nodemon -e ts,tsx,json,js,jsx --watch ./src --ignore ./dist --exec 'yarn build'",
+    "start": "nodemon -e ts,tsx,json,js,jsx --watch ./src --ignore ./dist --ignore src/utils/abi/typechain --exec 'yarn build'",
```

Comment:
@gsteenkamp89 This is a quick fix to stop the infinite loop. I'm not sure whether I prefer this or moving the typechain outputs to be outside of src. The latter is obviously more invasive. wdyt?